### PR TITLE
feat(cxp): add UI components for command contributions

### PR DESCRIPTION
**Pull request by chrismwendt**
image

### CODE REVIEW

File: app/github/inject.tsx [hunk 1]
```diff
@@ -201,6 +209,42 @@ function hideFileTree(): void {
...
+function injectCXPGlobalComponents({
+    cxpController,
+    extensionsContextController,
+}: {
+    cxpController: CXPController<ExtensionWithManifest, ConfigurationCascade<ConfigurationSubject, Settings>>
+    extensionsContextController: Controller<ConfigurationSubject, Settings>
+}): void {
+    const statusElem = document.createElement('div')
+    statusElem.id = 'cxp-global'
+    statusElem.style.position = 'fixed'
```
Comment (KattMingMing) [inline app/github/inject.tsx]: Can you move the inline styles to a classname

File: app/github/inject.tsx [hunk 2]
```diff
@@ -201,6 +209,42 @@ function hideFileTree(): void {
...
 
+function injectCXPGlobalComponents({
+    cxpController,
+    extensionsContextController,
+}: {
+    cxpController: CXPController<ExtensionWithManifest, ConfigurationCascade<ConfigurationSubject, Settings>>
+    extensionsContextController: Controller<ConfigurationSubject, Settings>
+}): void {
+    const statusElem = document.createElement('div')
+    statusElem.id = 'cxp-global'
```
Comment (KattMingMing) [inline app/github/inject.tsx]: Where is this ID used?

**Comment by KattMingMing** [comment]
@chrismwendt  can we get the command input to extend the full width of the popup (with some horizontal padding)

File: app/github/inject.tsx (hunk 1 above)
Comment (chrismwendt) [reply app/github/inject.tsx]: Moved to a classname (not pushed yet)

File: app/github/inject.tsx (hunk 2 above)
Comment (chrismwendt) [reply app/github/inject.tsx]: I don't think it's used at all - removed

**Comment by chrismwendt** [comment]
Yes, I'm tweaking the CSS right now

**Pull request merged**